### PR TITLE
[skip ci] Updated the document of examples/echo

### PR DESCRIPTION
**Pull request by kunihiko-t**
[skip ci] When I ran some examples on my local env, I noticed that there are errors in some commands.
So I updated the document.

### CODE REVIEW

File: examples/echo/README.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ USERNAME=YOUR_DOCKER_HUB_USERNAME
 ```
 # tagging
 docker run --rm -v "$PWD":/app treeder/bump patch
-docker tag $USERNAME/func-echo:latest $USERNAME/func-echo:`cat VERSION`
+docker tag iron/func-echo:latest $USERNAME/func-echo:`cat VERSION`
+docker tag $USERNAME/func-echo:`cat VERSION` $USERNAME/func-echo:latest
```
Comment (c0ze) [inline examples/echo/README.md]: Actually, fn tool now automatically tags latest, so probably this is redundant.

#639

Comment (kunihiko-t) [reply examples/echo/README.md]: I see.
I'll undo the changes.
Thanks.

Comment (c0ze) [reply examples/echo/README.md]: Thanks for the update!

Actually, now that latest tag is added automatically, we can update the docs to remove adding latest tag.

Would you mind updating to latest fn tool, confirming this behavior, and removing all reference to add latest tags from the documentation?

Comment (kunihiko-t) [reply examples/echo/README.md]: Yeah, I'll work on it.

You mean documents of examples should have this kind of "fn" style?
https://github.com/iron-io/functions/blob/0c3206deb5f8ebda5283662a271138a84ef8ecb1/examples/hello/go/README.md

Comment (c0ze) [reply examples/echo/README.md]: yes, if possible !

**Pull request merged**